diagnostics-app/src/app/app.component.spec.ts

```diff
@@ -13,12 +13,8 @@ import { AppComponent } from './app.component';
 describe('AppComponent', () => {
   beforeEach(async () => {
     await TestBed.configureTestingModule({
-      imports: [
-        RouterTestingModule
-      ],
-      declarations: [
-        AppComponent
-      ],
+      imports: [RouterTestingModule],
+      declarations: [AppComponent],
```

Comment:
do clean ups in a separate PR.

Reply:
This was done by the linter, we can keep this. Nothing's been changed here except linting.